tests/management/group/test_view.py

```diff
@@ -808,6 +809,16 @@ def test_update_default_group(self):
         response = client.put(url, test_data, format="json", **self.headers)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_update_custom_default_group(self):
+        """Test that Custom default group is protected from updates"""
+        customDefGroup = Group(name="customDefGroup", platform_default=True, system=False, tenant=self.tenant)
+        customDefGroup.save()
+        url = reverse("v1_management:group-detail", kwargs={"uuid": customDefGroup.uuid})
+        test_data = {"name": "new_name" + "_updated", "description": "new_description" + "_updated"}
+        client = APIClient()
+        response = client.put(url, test_data, format="json", **self.headers)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
```

Comment:
nice job 👍 I have small nitpick ... I would like to check not only status code but the text of the err message to be 100% sure that rbac returns 400 from correct condition, for example something like this
```
 expected_err_message = "Updating the name or description of 'Custom default group' is restricted"
 self.assertEqual(response.data.get("errors")[0].get("detail"), expected_err_message)
```
you can add it if you want but for now we can merge this PR